src/mudlet.cpp

```diff
@@ -1936,7 +1936,6 @@ void mudlet::createMapper( bool isToLoadDefaultMapFile )
 
     if( isToLoadDefaultMapFile && pHost->mpMap->mpRoomDB->getRoomIDList().isEmpty() )
     {
-        qDebug() << "mudlet::slot_mapper() - restore map case 3.";
```

Comment:
OK, will keep this line for you.

I'd prefer commented out code to be deleted, because it's still there in the Git history should you need to bring it back. How does that sound?